hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/BasicTypesAndCallbacksForAllDBsTest.java

```diff
@@ -381,7 +377,7 @@ public void testBigIntegerType(VertxTestContext context) {
 	}
 
 	@Test
-	@Disabled // Fail for MSSQL because the value changes before it's saved on the db. This also fails for ORM
+	@DisableFor(value = MYSQL, reason = "fails because the value changes before it's saved on the db. This also fails for ORM")
```

Comment:
It seems this test now works for all databases (the comment says `MSSQL` but you've disabled it for `MYSQL`)

Reply:
I haven't tested it for all JDK though

Reply:
fixed